Makefile

```diff
@@ -133,7 +136,7 @@ test-python-ci: create-db
 		tests/sentry \
 		tests/sentry_plugins \
 		tests/symbolicator \
-		--cov . --cov-report="xml:.artifacts/python.coverage.xml"
+		--cov . --cov-report="xml:.artifacts/python.coverage.xml" --cov-context=test
```

Comment:
Please check: cov-context=test has performance implications -- please make sure this doesn't regress test timings significantly

P1: CI time is a key engprod metric